etc/config/zig.amazon.properties

```diff
@@ -0,0 +1,15 @@
+compilers=&zig
+defaultCompiler=master
+
+group.zig.compilers=master
+group.zig.objdumper=/opt/compiler-explorer/gcc-8.1.0/bin/objdump
+group.zig.isSemVer=true
+group.zig.baseName=zig
+
+compiler.master.exe=/opt/compiler-explorer/zig-master/bin/zig
```

Comment:
Thankfuly I noticed this while checking the PR one last time before merge:
Can you change `master` to `trunk` as the compiler ID? It's how we refer to every other master version of the compilers.